app/models/nic/base.rb

```diff
@@ -202,6 +204,19 @@ def sync_name
       end
     end
 
+    def validate_host_taxonomy
+      return true if self.subnet.nil?
```

Comment:
It is a good idea, I was planning to do so, but as I said to Ohad before NICs are a special case - the NIC itself is not taxable.
Besides, `Host` object is not taxable, so it's not possible to properly join it.